### ISSUE TITLE
All notes in MuseSounds woodwinds, brass, strings, and vocals are slurred whenever possible

### ISSUE THREAD

**Issue by contrabassoon** [musescore/MuseScore#17602]
### Issue type

Muse Sounds bug

### Bug description

All notes are currently being slurred in woodwinds, brass, strings, and vocals if the interval is an octave or less (and not unison) and no articulation markings are applied

### Steps to reproduce

1. Create a new score containing any woodwind, brass, string, or vocal instrument and set the playback to use MuseSounds
2. Input at least two different back-to-back notes with leaps of less than an octave
3. Play back the notes

Expected behavior:
The notes are re-articluated

Actual behavior:
the notes are slurred despite the absence of a slur

### MuseScore Version

4.0.2.230651553

### Regression

No.

### Operating system

Windows 11

**Comment by msviolafangirl** [comment]
This one can be a little subjective and hard to pin down exactly, but I agree that there should be more definition in terms of the difference between slurred notes and non-slurred notes that otherwise have no other articulation marks. There is still some difference, just not enough

**Comment by msviolafangirl** [comment]
I agree that there should be more definition in terms of the difference between slurred notes and non-slurred notes that otherwise have no other articulation markings. I wouldn't say there's no difference at all, just not enough.

**Comment by LucySuccubus** [comment]
Not subjective at all. It is standard practice for all orchestral instruments (Woodwinds, Brass, Strings) to re-articulate notes when there are no slur markings or legato expression text. See the behavior of the Classic Phrasing sound flag for Musebrass as a reference. It is a good example of where to start when it comes to realizing the ideal playback expectations of the aforementioned Musesound instrument families.

**Comment by LucySuccubus** [comment]
My only critique of this report is to exclude vocals. Vocals are always legato (especially when the lyric the samples are "singing" is one continuous syllable).

**Comment by avenzi** [comment]
This is still an issue as of version 4.5.2, and makes these instruments inappropriate for many use cases. This is definitely **not a subjective distinction** especially for strings, as the difference between playing notes in one bow or separate bows is very pronounced, whereas there is **zero** difference in MuseSounds. 

https://github.com/user-attachments/assets/d7539e28-792e-4c16-9824-61246c244405

Even if you could hear a slight difference (which I cannot), these notes are clearly played in one bow.

**Comment by MarcSabatella** [comment]
The default is designed to cover cases where people simply don't include any articulations at all, which is pretty common.  But if you are using articulations carefully and wanting to hear the differentiation, add the "classic phrasing" sound flag.  Then you'll get result you would expect here.

**Comment by avenzi** [comment]
> The default is designed to cover cases where people simply don't include any articulations at all, which is pretty common. But if you are using articulations carefully and wanting to hear the differentiation, add the "classic phrasing" sound flag. Then you'll get result you would expect here.

The "classical phrasing" sound flag does **not** solve this issue, and in fact only creates more. The string sound is still that of notes being played in one bow, regardless of slurs. Note that this has nothing to do with the [Violin 1 portamento effect](https://github.com/musescore/MuseScore/issues/13878).

https://github.com/user-attachments/assets/e8a1d145-925f-4b3a-92aa-861f78c6dfea

This is what happens when using Violin 2 Solo (to avoid the [issues with Violin 1](https://github.com/musescore/MuseScore/issues/13878)). Note first that the dynamics are now completely off. If we then manually adjust the dynamics to compensate, we get the following:

https://github.com/user-attachments/assets/80ea43da-3fd3-4399-9f82-de274ff53711

Now ignoring the slight change in timbre and the glitch that occurs between measures due to the dynamics correction, we still hear that all notes are slurred together even when no slur is present.
